addons/l10n_mx/controllers/portal.py

```diff
@@ -0,0 +1,31 @@
+from itertools import count
+
+from odoo.addons.portal.controllers import portal
+from odoo.http import request
+
+class CustomerPortal(portal.CustomerPortal):
+
+    def _get_mandatory_fields(self):
+
```

Comment:
Why is there an empty line before the function's content? :eyes:
Let's fill it with `# EXTENDS 'portal'`, just to indicate we're extending a method.